platform/web/emscripten_helpers.py

```diff
@@ -38,7 +38,7 @@ def create_engine_file(env, target, source, externs):
 
 
 def create_template_zip(env, js, wasm, worker, side):
-    binary_name = "godot.tools" if env["tools"] else "godot"
+    binary_name = "godot.editor" if env.editor_build else "godot"
```

Comment:
@Faless Worth noting that I changed this too while at it. I've changed all cases of `"\.tools"` but I'm not sure that covers everything for the Web platform.